views/index.hbs

```diff
@@ -18,92 +18,67 @@
       <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Comment:
Where is the head of the document? You also don't import the css and firebase links